### PR TITLE
Feat/connect all pages

### PR DESCRIPTION
**Pull request by Parle648**
## Describe your changes

- Status management
   1. Customer Informed order status
   2. Transfer Locked order status
   3. Identity verified order status
   4. Boarding pass verified order status
   5. Baggage confirmed order status
   6. Customer confirmed order status
   7. Failed order status
- Order details page Connect to Be
- Connect All pages

## Issue ticket code (and/or) and link

- [Connect pages task](https://codecraftersintership.atlassian.net/browse/SCRUM-203?atlOrigin=eyJpIjoiMWQ1OTIyNDRkMTI2NGY5MWFlYmFkMTFiYTMwNjFiMjEiLCJwIjoiaiJ9)
- [Status management task](https://codecraftersintership.atlassian.net/browse/SCRUM-208?atlOrigin=eyJpIjoiZDE2ODZjYmVlN2M1NGQ2NDhjNzczMjA0ZjQyNjlkNWQiLCJwIjoiaiJ9)

## Loom videos
https://www.loom.com/share/02e05b13e45a498eb35724d356434a48?sid=222a7597-1fae-4506-a430-06b3c6630852
https://www.loom.com/share/b3d5fd8aa3f64832a27334c76215ee23?sid=d63be057-0660-4f35-af7a-506f97c4e19a

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [x] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/pages/PickUpTheOrder/BaggageCheck/BaggageCheckPage/components/CheckBaggageBlock/index.tsx [hunk 1]
```diff
@@ -12,29 +12,67 @@ import {
...
+        {luggages.length > 0 &&
+          luggages.map((luggage, index) => (
+            <Box key={luggage.id}>
+              <Typography sx={bagTitle}>
+                {`${luggage.luggage_type} bag ${index + 1}`}
+              </Typography>
+              <Typography sx={bagDetails}>
+                {t('checkBaggageBlock.bagDetails')}
+              </Typography>
+              <Box sx={{ display: 'flex', alignItems: 'center' }}>
```
Comment (oyakovytskyi) [inline src/pages/PickUpTheOrder/BaggageCheck/BaggageCheckPage/components/CheckBaggageBlock/index.tsx]: 1.3.4 inline styles prohibited

File: src/pages/PickUpTheOrder/BaggageCheck/BaggageCheckPage/useCalculateWeight.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+export const useCalculateWeight = (): UseCalculateWeightHook => {
+  const [weight, setWeight] = useState<number>(0);
+  const [isDisabled, setIsDisabled] = useState<boolean>(true);
+
+  const calculateWeight = (): void => {
+    let weightAmount = 0;
+    let hasEmptyInput = false;
+
+    document
+      .querySelectorAll('.MuiInputBase-input')
```
Comment (AlexeyBezrebriy) [inline src/pages/PickUpTheOrder/BaggageCheck/BaggageCheckPage/useCalculateWeight.ts]: Better to use useRef hook

**Review by oyakovytskyi — APPROVED** [review]

**Review by AlexeyBezrebriy — APPROVED** [review]

File: src/pages/PickUpTheOrder/BaggageCheck/BaggageCheckPage/components/CheckBaggageBlock/index.tsx (hunk 1 above)
Comment (Parle648) [reply src/pages/PickUpTheOrder/BaggageCheck/BaggageCheckPage/components/CheckBaggageBlock/index.tsx]: done

File: src/pages/PickUpTheOrder/BaggageCheck/BaggageCheckPage/useCalculateWeight.ts (hunk 2 above)
Comment (Parle648) [reply src/pages/PickUpTheOrder/BaggageCheck/BaggageCheckPage/useCalculateWeight.ts]: done

**Pull request merged**